lib/l10n/app_hi.arb

```diff
@@ -0,0 +1,30 @@
+{
+  "appTitle": "फ़ॉर्म",
+  "form": "फ़ॉर्म",
+  "nameQuestion": "आपका नाम क्या है?",
+  "nameFieldHint": "नाम*",
+  "nameError": "नाम कम से कम 3 वर्णों का होना चाहिए",
+  "birthDateQuestion": "आपका जन्म कब हुआ था?",
+  "birthDAteFieldHint": "जन्म तिथि*",
+  "birthDateError": "जन्म तिथि दर्ज करें",
+  "genderQuestion": "आप किस लिंग के रूप में पहचान करते हैं?",
+  "male": "पुरुष",
+  "female": "महिला",
+  "other": "अन्य",
+  "opinionQuestion": "आपकी राय जलवायु संकट के बारे में क्या है?",
+  "opinionFieldHint": "अपना संदेश दर्ज करें",
+  "originCountryQuestion": "आपका मूल देश क्या है?",
+  "originCountryDropdownHint": "देश का चयन करें",
+  "termsAndConditions": "मैं नियम और शर्तों से सहमत हूं",
+  "submit": "जमा करें",
+  "alertDialogContent": "फ़ॉर्म सफलतापूर्वक जमा किया गया!",
+  "close": "बंद करें",
+  "viewResponse": "प्रतिक्रिया देखें",
+  "yourResponse": "आपकी प्रतिक्रिया",
+  "goBack": "वापस जाएं",
+  "usa": "अमेरिका",
+  "india": "भारत",
+  "china": "चीन",
+  "japan": "जापान",
+  "russia": "रूस"
+}
```

Comment:
Was adding hindi translations a part of the exercise? Just curious to know.

Reply:
No. The task was to add localisations only. It was my curiosity to check if it's working as localisations should when I change the language of the system.

Reply:
Good work!